submissions/yhorodechna/side-menu-dom/null_style.css

```diff
@@ -0,0 +1,250 @@
+body {
```

Comment:
why do you have two style files?

Reply:
I always add null styles in a separate file. Now I added all styles in one file.